saleor/settings.py

```diff
@@ -400,7 +400,9 @@ def get_host():
         ('product_small', 'thumbnail__60x60'),
         ('product_small_2x', 'thumbnail__120x120'),
         ('product_list', 'thumbnail__255x255'),
-        ('product_list_2x', 'thumbnail__510x510')]}
+        ('product_list_2x', 'thumbnail__510x510')],
+    'background_images': [
+        ('background_image', 'thumbnail__960x320')]}
```

Comment:
Note that for now, only put the size `960x320` as their was no indications and seemed to be the best fit for large screens on a bootstrap container. And there is no lazyload yet on the image field. 

So, would we want to drop the CSS `background-image` and add support of a bigger size or do we keep that outside of this PR (probably better)?